src/Akka.Persistence.Sql/Journal/SqlWriteJournal.cs

```diff
@@ -137,13 +137,18 @@ private bool Initializing(object message)
         
         protected override bool ReceivePluginInternal(object message)
         {
-            if (message is not WriteFinished wf)
-                return false;
-
-            if (_writeInProgress.TryGetValue(wf.PersistenceId, out var latestPending) & (latestPending == wf.Future))
-                _writeInProgress.Remove(wf.PersistenceId);
-
-            return true;
+            switch (message)
+            {
+                case WriteFinished wf:
+                    if (_writeInProgress.TryGetValue(wf.PersistenceId, out var latestPending) & (latestPending == wf.Future))
+                        _writeInProgress.Remove(wf.PersistenceId);
+                    return true;
+                case IsInitialized:
```

Comment:
Should add a comment here explaining why this message exists

Reply:
Good idea

Reply:
Done